istio-control/istio-config/templates/deployment.yaml

```diff
@@ -20,20 +24,24 @@ spec:
         istio: galley
       annotations:
         sidecar.istio.io/inject: "false"
-        scheduler.alpha.kubernetes.io/critical-pod: ""
     spec:
       serviceAccountName: istio-galley-service-account
 {{- if .Values.global.priorityClassName }}
       priorityClassName: "{{ .Values.global.priorityClassName }}"
 {{- end }}
       containers:
         - name: galley
-          image: "{{ .Values.global.hub | default "gcr.io/istio-release"}}/{{ .Values.galley.image | default "galley" }}:{{ .Values.global.tag }}"
+{{- if contains "/" .Values.galley.image }}
+          image: "{{ .Values.galley.image }}"
+{{- else }}
+          image: "{{ .Values.global.hub }}/{{ .Values.galley.image | default "galley" }}:{{ .Values.global.tag }}"
+{{- end }}
           imagePullPolicy: {{ .Values.global.imagePullPolicy | default "Always" }}
           ports:
           - containerPort: 443
-          - containerPort: {{ .Values.global.monitoringPort | default "15014" }}
+          - containerPort: {{ .Values.global.monitoringPort }}
```

Comment:
Can we leave it ? monitoringPort is another setting we may deprecate, there is no reason to customize it or plan to test such customization.